classes/Console/Command/ClearCacheCommand.php

```diff
@@ -71,18 +71,22 @@ private function deleteFilesAndDirectoriesIn(string $path): int
     {
         $count = 0;
 
-        $filesAndDirectories = new \RecursiveIteratorIterator(
-            new \RecursiveDirectoryIterator(
-                $path,
-                \RecursiveDirectoryIterator::SKIP_DOTS
-            ),
-            \RecursiveIteratorIterator::CHILD_FIRST
-        );
-
-        foreach ($filesAndDirectories as $fileInfo) {
-            if ($this->delete($fileInfo)) {
-                ++$count;
+        try {
+            $filesAndDirectories = new \RecursiveIteratorIterator(
```

Comment:
Can we limit the `try`-`catch` block to this expression, and maybe just return in the `catch` block?

```diff
diff --git a/classes/Console/Command/ClearCacheCommand.php b/classes/Console/Command/ClearCacheCommand.php
index db37ca60..9a4cf60b 100644
--- a/classes/Console/Command/ClearCacheCommand.php
+++ b/classes/Console/Command/ClearCacheCommand.php
@@ -69,8 +69,6 @@ final class ClearCacheCommand extends Command

     private function deleteFilesAndDirectoriesIn(string $path): int
     {
-        $count = 0;
-
         try {
             $filesAndDirectories = new \RecursiveIteratorIterator(
                 new \RecursiveDirectoryIterator(
@@ -79,14 +77,16 @@ final class ClearCacheCommand extends Command
                 ),
                 \RecursiveIteratorIterator::CHILD_FIRST
             );
+        } catch (\UnexpectedValueException $exception) {
+            return 0;
+        }
+
+        $count = 0;

-            foreach ($filesAndDirectories as $fileInfo) {
-                if ($this->delete($fileInfo)) {
-                    ++$count;
-                }
+        foreach ($filesAndDirectories as $fileInfo) {
+            if ($this->delete($fileInfo)) {
+                ++$count;
             }
-        } catch (\UnexpectedValueException $e) {
-            // do nothing
         }

         return $count;
```